docs/netlify.toml

```diff
@@ -261,12 +261,12 @@
 
 [[redirects]]
   from = "/:version/integrations/smart-driver/"
-  to = "/preview/integrations/jdbc-driver/"
+  to = "/preview/reference/drivers/java/yugabyte-jdbc-reference/"
   force = true
 
 [[redirects]]
-  from = "/:version/integrations/*"
-  to = "/preview/integrations/:splat"
+  from = "/:version/integrations/*-driver/"
```

Comment:
This one should happily catch what the previous is doing. Try commenting out the redirect that starts on line 262 and make sure everything still works as intended.

Reply:
(And try a URL that matches `/:version/integrations/*` just to make sure the `-driver` is actually doing something! Sometimes regex implementations suck...)

Reply:
Here's what these two actually need to look like:

```toml
[[redirects]]
  from = "/:version/integrations/*-driver/"
  to = "/preview/reference/drivers/java/yugabyte-jdbc-reference/"
  force = true

[[redirects]]
  from = "/:version/integrations/*"
  to = "/preview/integrations/:splat"
  force = true
```

Reply:
Netlify was not recognizing the pattern. Modified it to be more specific and this works as expected. Will merge the PR.